src/main/java/mekanism/common/content/gear/mekasuit/ModuleMekaSuit.java

```diff
@@ -398,4 +404,19 @@ public void addHUDElements(List<HUDElement> list) {
             }
         }
     }
+    
+    public static class ModuleGeigerUnit extends ModuleMekaSuit {
+        private static final ResourceLocation icon = MekanismUtils.getResource(ResourceType.GUI_HUD, "geiger_counter.png");
+        @Override
+        public void addHUDElements(List<HUDElement> list) {
+            if (!isEnabled()) {
+                return;
+            }
+            {
+                double magnitude = MekanismAPI.getRadiationManager().getRadiationLevel(Minecraft.getInstance().player);HUDElement e = HUDElement.of(icon, UnitDisplayUtils.getDisplayShort(magnitude, RadiationUnit.SV, 3));
+                HUDColor color = magnitude < RadiationManager.MIN_MAGNITUDE ? HUDColor.REGULAR : (magnitude < 0.1 ? HUDColor.WARNING : HUDColor.DANGER);
+                list.add(e.color(color));
+            }
```

Comment:
When I said the else isn't needed I also meant the brackets that go with it aren't needed either. Also `HUDElement e` should be on the next line and not sharing the same line with `double magnitude`